### PR TITLE
Allow for knob specific mouse sensitivity via CKnobBase::setKnobRange()

### PR DESCRIPTION
**Pull request by raygit83**
In response to https://forums.steinberg.net/t/vstgui-knob-sensitivity/873343/2

### CODE REVIEW

**Comment by scheffle** [comment]
And if possible try to use the code style from VSTGUI. (Most easy way is to use git-clang-format if you're familiar with it).

**Comment by raygit83** [comment]
> Can you also add a get method, so it can be integrated into the UI Editor. Thanks.

Done.

File: vstgui/lib/controls/cknob.h [hunk 1]
```diff
@@ -33,6 +33,9 @@ class CKnobBase : public CControl, protected CMouseWheelEditingSupport
 
 	virtual CCoord getInsetValue () const { return inset; }
 	virtual void setInsetValue (CCoord val) { inset = val; }
+
+	virtual void setKnobRange (float val) { if(val > 0.f) knobRange = val; }
```
Comment (nickdowell) [inline vstgui/lib/controls/cknob.h:37]: There should be a space between the `if` and `(` ;-)

```suggestion
	virtual void setKnobRange (float val) { if (val > 0.f) knobRange = val; }
```

Comment (scheffle) [reply vstgui/lib/controls/cknob.h:37]: @raygit83 this one I meant where the code style was wrong ;-)

**Comment by scheffle** [comment]
Ah, and please base the change on the develop branch. That's where changes should be made. Only bugfixes and releases should go into the master branch.

**Comment by raygit83** [comment]
> Ah, and please base the change on the develop branch. That's where changes should be made. Only bugfixes and releases should go into the master branch.

Got it...done.

**Comment by scheffle** [comment]
Done with #312

**Pull request closed without merging**